### DISCUSSION TITLE
[@types/numjs] mod function's arg is invalid

### DISCUSSION THREAD

**Discussion by kmtszky** [DefinitelyTyped/DefinitelyTyped#68722] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a1980b0af2c54994fe9f217ace09afa59c2cf72e/types/numjs/index.d.ts#L279

I think it should be 
```
export function mod<T = number>(x1: NjParam<T>, x2: NjParam<T>): NdArray<T>;
```
like here.
https://github.com/nicolaspanel/numjs/blob/b0b71237595bf6c9b57ade9e0e5dc720f7e1545b/src/index.js#L206-L208

Please check them.

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Fixed in #68723
